src/screens/AccountScreens/ProfileScreen/index.js

```diff
@@ -1,11 +1,56 @@
 import React from 'react';
 import { styles } from './style';
-import { View, Text } from 'react-native';
+import { View, Text, Image, } from 'react-native';
+import { Button, Input, Loading } from '../../../components';
+import { useAuthStore } from '../../../shared/zustand/auth';
+import { THEME_COLORS } from '../../../components/theme';
+
+
 
 const Profile = () => {
+  const { isLoading } = useAuthStore();
+  if (isLoading) {
+    return <Loading />;
+  }
   return (
     <View style={styles.container}>
-      <Text style={{fontSize: 30,}}>Profile Page</Text>
+      <Image
```

Comment:
We should make a standalone component, can name it as `ProfileImage` or something similar. This component can be reused in the future (e.g., on the team member page), and making it a more generic component can be more handy compared to manually styling each time by using the `Image` component provided by React Native. We could also add some custom props if we make it a standalone component.